lib/mergometer/report.rb

```diff
@@ -46,13 +46,19 @@ def entries
       end
 
       def filter
-        "base:master repo:#{repo} type:pr is:merged"
+        "base:master #{repo_query} type:pr is:merged"
       end
 
       def fields_to_preload
         []
       end
 
+      def repo_query
+        @_repo_query = repo.split(",").map do |r|
+          "repo:#{r} "
```

Comment:
Nit: is there an extra space here? Could maybe add it to `join(" ")` instead?

Reply:
Done! @balvig